bpfd/src/storage.rs

```diff
@@ -42,6 +57,7 @@ struct CsiIdentity {
 
 struct CsiNode {
     node_id: String,
+    tx: Arc<Mutex<Sender<Command>>>,
 }
```

Comment:
```suggestion
    tx: Sender<Command>,
```

This doesn't need to be wrapped in `Arc<Mutex<>>` as the channel is MPSC - there are supposed to be multiple producers. That's true for BpfLoader also.

Reply:
Done and also fixed for bpfLoader in a separate commit 👍